### ISSUE TITLE
[gatsby-theme-minimal-blog]: All code blocks missing in RSS feed

### ISSUE THREAD

**Issue by sidm1983** [LekoArts/gatsby-themes#299]
## Description

All code blocks in blog posts are not being rendered in the `content:encoded` section of the item in the `/rss.xml` feed. This same issue can be seen on the themes demo site too. If you compare the content on `https://minimal-blog.lekoarts.de/fantastic-beasts-and-where-to-find-them` to the relevant item in the RSS feed (`https://minimal-blog.lekoarts.de/rss.xml`), you will see that all the code block `<div>` elements are missing from the RSS feed.

### Steps to reproduce

1. Create a new site using the minimal blog theme.
1. Create a post with one or more code blocks in it.
1. Go to the RSS feed (/rss.xml) and find the post in the feed.
1. All the code blocks will be missing from the post content within the RSS feed.

### Expected result

If a post has code blocks, then all code block `<div>` elements should be correctly rendered within the `content:encoded` section of an item in the RSS feed.

### Actual result

All code blocks within a post are never rendered in the RSS feed.

### Environment

```
  System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz
  Binaries:
    Node: 10.15.3 - C:\Program Files\nodejs\node.EXE
    npm: 6.4.1 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 3.7.1
  Browsers:
    Edge: 44.18362.449.0
  npmPackages:
    gatsby: ^2.13.3 => 2.19.17
    gatsby-plugin-google-analytics: ^2.1.4 => 2.1.35
    gatsby-plugin-manifest: ^2.2.3 => 2.2.41
    gatsby-plugin-netlify: ^2.1.3 => 2.1.32
    gatsby-plugin-offline: ^2.2.4 => 2.2.10
    gatsby-plugin-sitemap: ^2.2.19 => 2.2.27
    gatsby-plugin-webpack-bundle-analyser-v2: ^1.1.8 => 1.1.8
```

**Comment by sidm1983** [comment]
@LekoArts, any ideas or thoughts on this issue?

**Comment by LekoArts** [comment]
This is due to me using `@loadable/component` for the code blocks. I'll try to do SSR on them, tracking it here: https://github.com/hector-del-rio/gatsby-plugin-loadable-components-ssr/issues/3

**Comment by sidm1983** [comment]
Thank you for looking into it and for opening an issue ticket for it with the loadable project team. I have subscribed to notifications on that issue thread. Hopefully, there is a solution to it soon.

**Comment by sidm1983** [comment]
@LekoArts Any thoughts on this now that the dev from loadable has responded and closed the issue?

**Comment by sidm1983** [comment]
Hi @LekoArts, any ideas on this issue? Just wondering if you are still working on a solution or whether there are any usable workarounds. Any help would be greatly appreciated. Thank you.

**Comment by LekoArts** [comment]
So I investigated that issue a bit, tested raw MDX, MDX with `prism-react-renderer` and lastly with `@loadable/component` and compared the last two with the webpack-bundle-analyzer. I initially added is due to the weight of the packages but since https://github.com/gatsbyjs/gatsby/pull/22253 was added it's not really necessary anymore or is more in the realms of mini-optimization. So I removed the lazy loading part again.
